src/resources/resources.response.ts

```diff
@@ -0,0 +1,45 @@
+import { ApiProperty } from "@nestjs/swagger";
+
+export class TeamResourceResponse {
+    @ApiProperty({ example: 1 })
+    id: number;
+
+    @ApiProperty({ example: 1 })
+    teamMemberId: number;
+
+    @ApiProperty({ example: "https://www.github.com/" })
+    url: string;
+
+    @ApiProperty({ example: "Github" })
+    title: string;
+
+    @ApiProperty({ example: "2023-11-30T06:47:11.694Z" })
+    createdAt: Date;
+
+    @ApiProperty({ example: "2023-11-30T06:47:11.694Z" })
+    updatedAt: Date;
+}
+
+class MemberNameAndAvatar {
+    @ApiProperty({ example: "Jessica" })
+    firstName: string;
+
+    @ApiProperty({ example: "Williamson" })
+    lastName: string;
+
+    @ApiProperty({
+        example:
+            "https://gravatar.com/avatar/3bfaef00e02a22f99e17c66e7a9fdd31?s=400&d=robohash&r=x",
+    })
+    avatar: string;
+}
+
+class AddedBy {
+    @ApiProperty()
+    member: MemberNameAndAvatar;
+}
+
+export class TeamResourceAddedByResponse extends TeamResourceResponse {
+    @ApiProperty({ isArray: true })
```

Comment:
Hey shouldn't this not be an array since only one person can be listed as addedBy?

![image](https://github.com/chingu-x/chingu-dashboard-be/assets/104289709/f97d1c92-a6cd-4edb-b5a5-385949467b30)

Reply:
Yeah, I guess I saw that nested object and my mind went "array!" for some reason😄